plugin/src/test/java/io/jenkins/plugins/casc/SchemaGenerationTest.java

```diff
@@ -68,11 +47,15 @@ public void invalidSchemaShouldNotSucceed() throws Exception {
         JSONObject jsonSchema = new JSONObject(
             new JSONTokener(prettyJsonString));
         String yamlStringContents = Util.toStringFromYamlFile(this, "invalidSchemaConfig.yml");
-        System.out.println(Util.convertToJson(yamlStringContents));
         JSONObject jsonSubject = new JSONObject(
             new JSONTokener(Util.convertToJson(yamlStringContents)));
         Schema schema = SchemaLoader.load(jsonSchema);
-        schema.validate(jsonSubject);
+        try {
+            schema.validate(jsonSubject);
```

Comment:
@casz The basic schema is validated easily.However if the yml becomes a bit more comples the validation fails.